src/classes/applicant.py

```diff
@@ -0,0 +1,86 @@
+from uuid import UUID, uuid4
+from datetime import date
+from user import User
+
+
+class Applicant(User):
+
+    def __init__(
+            self,
+            first_name: str,
+            last_name: str,
+            date_of_birth: date,
+            street: str,
+            postal_code: int,
+            city: str,
+            email: str,
+            phone_number: str,
+            rating: float = 0.0
+    ) -> None:
+        super().__init__(
+            first_name=first_name,
+            last_name=last_name
+        )
+        self._date_of_birth = date_of_birth
+        self._street = street
+        self._postal_code = postal_code
+        self._city = city
+        self._email = email
+        self._phone_number = phone_number
+        self._rating = rating
+
```

Comment:
Missed out Attributes:

        self._curriculum_vitae = CurriculumVitae
        self._cover_letters = []
        self._skills = []
        self._applications = []

Reply:
These Attributes consists of the Classes: Skill, CoverLetter, CurriculumVitae and ApplicantApplication

make sure to also create the getter an setter functions for them